docker-compose.yaml

```diff
@@ -11,23 +13,43 @@ services:
       - 3000:3000
     environment:
       - DATABASE_URL="postgresql://postgres:postgres@postgres:5432/merch"
-    networks:
-      - app_network
-
+    depends_on:
+      - postgres
+  web-dev:
+    container_name: web-dev
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - .:/app
+      # NOTE(luisd): this is the main downsize of using volumes and pnpm
+      # pnpm requires a store, but because it's inside of a container we can't access
+      # the default (if any) store, so we override the root node_modules and the local store
+      - /app/node_modules
```

Comment:
This is very sus, on Windows, Next.js will just not work because the dev node_modules has the windows version of SWC but the container will be running linux as the OS.